pkg/datapath/linux/probes/probes.go

```diff
@@ -60,6 +60,13 @@ func (kp KernelParam) Module() bool {
 	return kp == "m"
 }
 
+// Holds information about kernelOptions being used by CLI kernel-check
```

Comment:
```suggestion
// kernelOption holds information about kernel parameters to probe.
```
It is not restricted to CLI, since we are using them in `SystemConfigProbes` as well.